jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml

```diff
@@ -237,3 +237,10 @@
         job-name: ci-kubernetes-e2e-gce-ubuntu-1-6-node-serial
         repo-name: k8s.io/kubernetes
         timeout: 240
+
+    - kubernetes-release:
+        branch: master
+        commit-frequency: 'H/2 * * * *'
```

Comment:
Thanks.  BTW, what is the format of the frequency field.  It looks cron-like, but...